releases/release-7.4.0.md

```diff
@@ -0,0 +1,186 @@
+---
+title: TiDB 7.4.0 Release Notes
+summary: Learn about the new features, compatibility changes, improvements, and bug fixes in TiDB 7.4.0.
+---
+
+# TiDB 7.4.0 Release Notes
+
+Release date: xx xx, 2023
+
+TiDB version: 7.4.0
+
+Quick access: [Quick start](https://docs.pingcap.com/tidb/v7.4/quick-start-with-tidb) | [Installation packages](https://www.pingcap.com/download/?version=v7.4.0#version-list)
+
+7.4.0 introduces the following key features and improvements:
+
+<!-- key feature placeholder-->
+
+## Feature details
+
+### Performance
+
+- note 1
+
+- note 2
+
+### Reliability
+
+- note 1
+
+- note 2
+
+### SQL
+
+- note 1
+
+- note 2
+
+### DB operations
+
+- note 1
+
+- note 2
+
+### Observability
+
+- note 1
+
+- note 2
+
+### Data migration
+
+- note 1
+
+- note 2
```

Comment:
```suggestion
* Support real-time update of checkpoint for incremental data validation [#issue号](链接) @[lichunzhu](https://github.com/lichunzhu) **tw@ran-huang** <!--1496-->

    Before v7.4.0, the incremental data validation feature is used to determine whether the data replication from DM to downstream is consistent with upstream. This is used as the basis for cutting over business traffic from upstream database to TiDB. However, due to various factors such as replication delay and waiting for re-validation of inconsistent data, the incremental validation checkpoint must be refreshed every few minutes. This is unacceptable for some business scenarios where the cutover time is only a few tens of seconds.

    With the introduction of real-time updating of checkpoint for incremental data validation, you can pass in the binlog position filled in by the upstream database. Once the incremental validation program detects this binlog position in memory, it will immediately refresh the checkpoint instead of refreshing it every few minutes. Therefore, you can quickly perform cut-off operations based on this immediately returned checkpoint.

    For details, see [user document](链接)。
    
    
```